### DISCUSSION TITLE
Welcome to Bootstrap-Flask Discussions!

### DISCUSSION THREAD

**Discussion by greyli** [helloflask/bootstrap-flask#214] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by PanderMusubi** [comment]
My (nick)name is @PanderMusubi and I am an information analyst who uses Python. Bootstrap-Flask allows me to quickly develop responsive dynamic websites from Python's realm, without having to deal with CSS or Javascript. I contribute code to this community and appreciate how @greyli handles issues and pull requests.

**Comment by xprimate** [comment]
Hello guys.
Please I am not clear on how to implement some of the form field listed in the screen-shot below(from the demo application)
I have created the the form.py file and copied the class codes.
Specifically, I want to use the checkbox form.
Kindly help.
![Screenshot 2022-10-17 at 01-03-47 Bootstrap-Flask Demo Application](https://user-images.githubusercontent.com/17166016/196094010-70f58e46-6ad3-4955-a4f2-b0458d00c31e.png)
o it but it is not available in the template.

**Reply by PanderMusubi** [reply]
When you're not yet familiar enough with Flask etc., start with the example application and remove step by step what you don't need. The info you have listed is not enough to help and the proposed way will help to understand this this works better.

**Reply by xprimate** [reply]
Got it now; create the form class, instantiate and pass it in view function, then render the template together with the class object.
Thanks.

**Comment by natcobbinah** [comment]
Bonjour everyone, I am Cobbinah Nathaniel, and a Software Engineer, but not defined by a specific tech-stack although I gravitate towards (Python and Java), as I believe with a strong foundation in Computer Science and engineering picking up the tools and language required for the job shouldn't be a problem. Using Bootstrap-Flask in my flask projects helps to remove a lot of boiler-plate codes with its **macros** designed to make rendering of bootstrap elements easier and make templates less cluttered.
